packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -74,6 +74,7 @@ export class DataStores implements IDisposable {
             (id: string, createParam: CreateChildSummarizerNodeParam)  => CreateChildSummarizerNodeFn,
         private readonly deleteChildSummarizerNodeFn: (id: string) => void,
         baseLogger: ITelemetryBaseLogger,
+        private readonly metadata: IContainerRuntimeMetadata,
```

Comment:
This doesn't have to be a member of the class since you are only using it in the constructor.